contracts/pool/src/lib.rs

```diff
@@ -146,6 +147,10 @@ impl LendingPoolTrait for LendingPool {
         init_reserve(&env, &asset, &input)
     }
 
+    fn change_decimals(env: Env, asset: Address, decimals: u32) -> Result<(), Error> {
```

Comment:
Let's replace the prefix with "set_" since we use such convention across the code.